src/elements/common/_animations.scss

```diff
@@ -0,0 +1,27 @@
+@keyframes fade-in {
```

Comment:
Could we consider moving this up to `styles/common` so shared animations can be use both in existing elements and the other features/components?